src/main/java/seedu/address/ui/UiManager.java

```diff
@@ -66,7 +66,7 @@ void showAlertDialogAndWait(Alert.AlertType type, String title, String headerTex
     private static void showAlertDialogAndWait(Stage owner, AlertType type, String title, String headerText,
                                                String contentText) {
         final Alert alert = new Alert(type);
-        alert.getDialogPane().getStylesheets().add("view/DarkTheme.css");
+        alert.getDialogPane().getStylesheets().add("view/LightTheme.css");
```

Comment:
Will the user be able to change back to DarkTheme?

Reply:
Nope, if you guys are agreeable with the new LightTheme Ui, I will go ahead and delete DarkTheme.css

<img width="1788" alt="Screenshot 2021-03-24 at 7 00 43 PM" src="https://user-images.githubusercontent.com/70088064/112299963-49dadb80-8cd3-11eb-89a8-4b2dd08f1d44.png">

Reply:
izzit possible to have something to toggle between light and dark themes?

Reply:
I would say it's not impossible but the purpose of adopting a light theme was to allow the greying out of expired property and appointments to be somewhat observable yet inconspicuous. The grey effect in the already Black/Grey DarkTheme would probably not achieve the same effect. However, if you guys are keen on a dark theme, I can consider a DarkTheme design in a subsequent update- maybe keep the DarkTheme.css file for now

Reply:
i see